scanpy/testing/_pytest/fixtures/data.py

```diff
@@ -2,17 +2,37 @@
 
 from __future__ import annotations
 
+import tempfile
```

Comment:
don’t use that in tests, please. There’s the `tmp_path` and `tmp_path_factory` fixtures: https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
I think `tmp_path` is not scoped for session though?

Reply:
For now I scoped the `backed_adata` to `function` but I don't really agree with this.

Reply:
`tmp_path_factory` is session scoped. I guess the docs could be improved to point out that the directories it creates are also session scoped.